clj-refactor.el

```diff
@@ -2826,6 +2826,7 @@ See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-find-usages"
 
 See: https://github.com/clojure-emacs/clj-refactor.el/wiki/cljr-rename-symbol"
   (interactive)
+  (if (buffer-narrowed-p) (error "Cannot rename symbols while narrowing is in effect"))
```

Comment:
Think this needs to be `when` or the linter is going to complain about an `if` clause without corresponding `else` branch.

Reply:
Done.